service/src/main/java/uk/gov/hmcts/reform/fpl/model/HearingBooking.java

```diff
@@ -3,22 +3,34 @@
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import uk.gov.hmcts.reform.fpl.validation.groups.HearingBookingDetailsGroup;
+import uk.gov.hmcts.reform.fpl.validation.interfaces.time.HasEndDateAfterStartDate;
+import uk.gov.hmcts.reform.fpl.validation.interfaces.time.HasSetTime;
 
-import java.time.LocalDate;
+import java.time.LocalDateTime;
 import java.util.List;
+import javax.validation.constraints.Future;
 
 @Data
 @Builder
 @AllArgsConstructor
+@HasEndDateAfterStartDate(groups = HearingBookingDetailsGroup.class)
 public class HearingBooking {
     private final String type;
     private final String typeDetails;
     private final String venue;
-    private final LocalDate date;
-    private final String preHearingAttendance;
-    private final String time;
+    @HasSetTime(message = "Enter a valid start time", groups = HearingBookingDetailsGroup.class)
```

Comment:
Is content happy with that error message? How it is obvious what's valid time?

Reply:
This is the message that was on the ticket so I assume so